package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@observation.org/react-native-components",
-  "version": "1.24.0",
+  "version": "1.26.0",
```

Comment:
Ik heb hier `1.26.0` van gemaakt omdat ik PR #31 zag met versie `1.25.0`

Reply:
Nee, dat is niet goed. Als deze PR eerder wordt goedgekeurd en gemergd, dan hebben we een versienummer overgeslagen. Daarom altijd de volgende versie ten opzichte van develop hanteren. En mocht de andere PR eerst gaan, dan krijg je vanzelf een conflict op het versienummer, en kun je dit vervolgens bijwerken.

Reply:
Ah goed om te weten!

Dan pas ik het aan!